packages/docs/docs/getting-started/flow.mdx

```diff
@@ -98,36 +98,128 @@ function. Generators like this are called tweens, because they animate
 be**tween** two values. You can read more about them in the
 [tweening](/docs/tweening) section.
 
+## Flow Generators
+
 Another kind of generators are _flow generators_. They take one or more
 generators as their input and combine them together. We've mentioned the `all()`
 generator in the quickstart section, there's a few more:
 
-## `all`
+### `all`
 
 <ApiSnippet url={'/api/core/flow#all'} />
 <hr />
 
-## `any`
+### `any`
 
 <ApiSnippet url={'/api/core/flow#any'} />
 <hr />
 
-## `chain`
+### `chain`
 
 <ApiSnippet url={'/api/core/flow#chain'} />
 <hr />
 
-## `delay`
+### `delay`
 
 <ApiSnippet url={'/api/core/flow#delay'} />
 <hr />
 
-## `sequence`
+### `sequence`
 
 <ApiSnippet url={'/api/core/flow#sequence'} />
 <hr />
 
-## `loop`
+### `loop`
 
 <ApiSnippet url={'/api/core/flow#loop'} />
 <hr />
+
+## Looping
+
+There are many ways to animate multiple objects. Here are some examples. Try
+using them in the below editor.
+
+```tsx editor
+import {ThreadGenerator} from '@motion-canvas/core/lib/threading';
+import {makeRef} from '@motion-canvas/core/lib/utils';
+import {waitFor} from '@motion-canvas/core/lib/flow';
+import {Rect} from '@motion-canvas/2d/lib/components';
+
+export default makeScene2D(function* (view) {
+  const rects = [];
+
+  // Create some rects
+  for (let i = 0; i < 5; i++) {
+    view.add(
+      <Rect
+        ref={makeRef(rects, i)}
+        width={100}
+        height={100}
+        x={-250 + 125 * i}
+        fill="#88C0D0"
+        radius={10}
+      />,
+    );
+  }
+
+  yield* waitFor(1);
+
+  // Animate them
+  yield* all(
+    ...rects.map(rect => rect.position.y(100, 1).to(-100, 2).to(0, 1)),
+  );
+});
+```
+
+### Using `Array.map` and `all`
+
+This is one of the most elegant ways to do simple tweens, but requires nesting
+`all` to do multiple tweens on an object since the `map` callback must return a
+`ThreadGenerator`.
+
+```tsx
+yield *
+  all(
+    ...rects.map(rect =>
+      // No yield or anything; we return this generator and deal with it outside
+      rect.position.y(100, 1).to(-100, 2).to(0, 1),
+    ),
+  );
+```
+
+<hr />
```

Comment:
I don't think the `<hr />`s here are necessary.
I use them when including `ApiSnippet`s because they are visually busy and blend together otherwise. 
But here it looks good without them.